File: android/src/main/java/com/onegini/mobile/view/handlers/PinNotificationObserver.java
//@todo Later will be transferred to RN Wrapper
package com.onegini.mobile.view.handlers;

import androidx.annotation.NonNull;

import com.onegini.mobile.Constants;

import javax.annotation.Nullable;

public interface PinNotificationObserver {

  void onNotify(String type, @NonNull Constants.PinFlow flow);

  void onError(String message);
}

File: android/src/main/java/com/onegini/mobile/view/handlers/PinNotificationObserver.kt
//@todo Later will be transferred to RN Wrapper
package com.onegini.mobile.view.handlers

import com.onegini.mobile.Constants.PinFlow
import com.onegini.mobile.sdk.android.handlers.error.OneginiError

interface PinNotificationObserver {
    fun onNotify(event: String, flow: PinFlow)
    fun onError(error: OneginiError?)
}